Vagrantfile

```diff
@@ -39,21 +39,20 @@ begin
     end
   end
 
-
   if data_hash['vagrant'] && data_hash['vagrant']['config_paths']
     data_hash['vagrant']['config_paths'].map do |default_config_file_path|
       overide_config_file_path = default_config_file_path.gsub(/default\.json$/, "overide.json")
-      if File.exist?(default_config_file_path)
-        default_config_file = File.read(default_config_file_path)
+
+      if File.exist?(File.dirname(__FILE__) + '/' + default_config_file_path)
+        default_config_file = File.read(File.dirname(__FILE__) + '/' + default_config_file_path)
         parsing_file = default_config_file_path
         project_config_hash = JSON.parse(default_config_file)
       else
-        puts red('Error read file ' + default_config_file_path + '. Please check if that file exist. Exiting.' )
-        exit!
+        puts yellow('Cannot read file ' + default_config_file_path + '. Please check if that file exist.' )
```

Comment:
because this is optional, let's put it into INFO level and don't require users to do anything.

puts "[teracy-dev][INFO]: #{default_config_file_path} not found, make sure this is intended."